server/controllers/ticket/comment.php

```diff
@@ -43,7 +43,7 @@ class CommentController extends Controller {
 
     public function validations() {
         $this->session = Session::getInstance();
```

Comment:
Let's take this opportunity to remove this line, and the variable `session` (line 41) from the class.